ytmusicapi/mixins/browsing.py

```diff
@@ -711,3 +710,41 @@ def get_signatureTimestamp(self, url: str = None) -> int:
             raise Exception("Unable to identify the signatureTimestamp.")
 
         return int(match.group(1))
+
+    def get_tasteprofiles(self) -> Dict:
+        """
+        Fetches all artists from taste profile (music.youtube.com/tasteprofile)
+
+        :return: Dictionary with artist and their selection & impression value
+
+        Example::
+        {
+            "Drake": {
+                "selectionValue": "tastebuilder_selection=/m/05mt_q",
+                "impressionValue": "tastebuilder_impression=/m/05mt_q"
+            }
+        }
+        """
+        response = self._send_request('browse', {'browseId': "FEmusic_tastebuilder"})
+        return parse_tasteprofiles(response)
+
+    def set_tasteprofile(self, *artists: str) -> None:
```

Comment:
Conventionally ytmusicapi uses List[str] for multiple str values, see for example here: https://github.com/sigma67/ytmusicapi/blob/d66f20aa13ce2117fce7abcadbb9ae288bd352e7/ytmusicapi/mixins/playlists.py#L239-L241

The function should return the YouTube Music response or error, not `None`.